tests/kafkatest/tests/streams/streams_broker_down_resilience_test.py

```diff
@@ -48,8 +50,6 @@ def test_streams_resilient_to_broker_down(self):
         processor = StreamsBrokerDownResilienceService(self.test_context, self.kafka, self.get_configs())
         processor.start()
 
-        # until KIP-91 is merged we'll only send 5 messages to assert Kafka Streams is running before taking the broker down
-        # After KIP-91 is merged we'll continue to send messages the duration of the test
         self.assert_produce_consume(self.inputTopic,
```

Comment:
I'm not breaking the produce/consume assertion here as there is only one streams client in this test.